src/routes/login/index.tsx

```diff
@@ -50,7 +55,12 @@ class Login extends Component<{}, LoginState> {
               onInput={linkState(this, 'password')}
               value={state.password}
             />
-            <Button type="submit" value="Login" />
+            <div class={buttons}>
+              <Button type="submit" value="Login" />
+              <Button type="button" onClick={this.handleRegister}>
```

Comment:
Why do you have a `type` at all? If you don't pass a `type`, the `<Button/>` component will become a `<button/>` element instead of a `<input type="submit"/>`

Reply:
It only allowed the type to be 'submit' or null.

Reply:
Right, that's why you don't need to specify a type at all. That is intended behavior. I don't see a reason to ever use `<input type="button"/>` because `<button>` makes a lot more sense

Reply:
Yeah but if I don't specify type="button" the default type is submit...

Reply:
So I actually can't remove it

Reply:
Maybe it should say type="button" if type is null on the Button component?

Reply:
If you don't specify `type="button"` then it should become a `<button/>` element, without a `type` prop

Reply:
Yes! But the button element without a type prop defaults to submit behavior which isn't what I want you absolute potate.